Simplified docs (#155)
LucidtechAI/cradl-docs: docs/get-started/parse-invoices-and-receipts.mdx

</TabItem>

<TabItem value="cli" label="Using the CLI">

:::info
You need to have the CLI installed. Follow the instructions [here](../reference/cli) to install the CLI.
:::

In order to parse an invoice in Cradl we need to make two API calls, one that uploads a 
[document](../concepts/documents) and another that parses it and creates [predictions](../concepts/predictions). The 
second API call requires you to provide a `modelId`. If you are using one of our pre-trained models they are 
`las:organization:cradl/las:model:invoice` and `las:organization:cradl/las:model:receipt` for the invoice and receipt 
models respectively. You can use these as a quick test, but we recommend training your own customized model for 
production usage. Read more about how to train a high accuracy model 
[here](../get-started/train-high-accuracy-models-on-large-datasets)

```bash
las documents create <your invoice.pdf>
las predictions create <documentId> las:organization:cradl/las:model:invoice
```

Congratulations, you have now successfully parsed your first invoice using Cradl. The JSON response body should have a 

> list of predictions on your uploaded document and the payload should look similar to the following.

Reviewer: ```suggestion
list of predictions for your uploaded document. The JSON payload should look similar to the following.
```

```json
{
  "predictionId": "las:prediction:...",
  "documentId": "las:document:...",
  "modelId": "las:organization:cradl/las:model:invoice",
  "predictions": [
    {
      "label": "vat_amount",
      "page": 1,
      "value": "75.72",
      "confidence": 0.9975606850585552
    },
    {
      "label": "total_amount",
      "page": 0,
      "value": "369.00",
      "confidence": 0.9931743085807881
    }
  ]
}
```

</TabItem>
<TabItem value="postman" label="Using Postman">

:::tip
Check out our Postman Collection [here](https://www.postman.com/cradlai).
:::

<h3> Log in to Cradl and create API credentials </h3>

> To begin you'll need to [log in to Cradl](https://app.cradl.ai/login?redirect=login&utm_campaign=docs) and create an 

Reviewer: ```suggestion
First of all, [log in to Cradl](https://app.cradl.ai/login?redirect=login&utm_campaign=docs) and create an 
```
